/*
 * This file was automatically generated by EvoSuite
 * Tue Sep 24 17:05:42 GMT 2024
 */


import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true) 
public class Ex1_ESTest extends Ex1_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Ex1.ex1(3486, 3486, 3486, 3486);
      Ex1.ex1(3486, 0, 0, 2);
      Ex1.ex1(1696, 0, 1696, 2);
      // Undeclared exception!
      Ex1.ex1(1696, 1, 0, 1);
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      Ex1.ex1(131, (-2140), (-2140), (-1));
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      Ex1.ex1((-1121), 1691, 1691, (-4589));
  }

  @Test(timeout = 4000)
  public void test3()  throws Throwable  {
      Ex1 ex1_0 = new Ex1();
  }
}
